### ISSUE TITLE
pkg_resources.DistributionNotFound: The 'webrtcvad' distribution was not found and is required by the application

### ISSUE THREAD

**Issue by elpimous** [pyinstaller/pyinstaller#3357]
sys : ubuntu 16
python 2.7
pyinstaller 3.3.1

The lib in fault : "webrtcvad"

Hello all, trying to make a unike file from my package, but I encounter a lib error !

my code : 
```
import webrtcvad
import collections
import contextlib
import os
import signal
import pyaudio
import wave
import scipy.io.wavfile as wav
from datetime import datetime
```

with python -v
```
import webrtcvad
import pkg_resources._vendor.packaging.markers # precompiled from /home/nvidia/.local/lib/python2.7/site-packages/pkg_resources/_vendor/packaging/markers.pyc
dlopen("/usr/local/lib/python2.7/dist-packages/_webrtcvad.so", 2);
import _webrtcvad # dynamically loaded from /usr/local/lib/python2.7/dist-packages/_webrtcvad.so

>>> webrtcvad.__version__
'2.0.10'
```

The compile worked ok !

But when lanching : here is the error: 

```
./survey_client 
Traceback (most recent call last):
  File "survey_client.py", line 15, in <module>
  File "/tmp/pip-build-etsESU/pyinstaller/PyInstaller/loader/pyimod03_importers.py", line 396, in load_module
  File "vader.py", line 8, in <module>
  File "pkg_resources/__init__.py", line 564, in get_distribution
  File "pkg_resources/__init__.py", line 436, in get_provider
  File "pkg_resources/__init__.py", line 984, in require
  File "pkg_resources/__init__.py", line 870, in resolve
pkg_resources.DistributionNotFound: The 'webrtcvad' distribution was not found and is required by the application
[6344] Failed to execute script survey_client
```

an idea ?
I read lot of answers, but didn't find the solution.
Tried hook, but perhaps badly !
Thanks all

**Comment by htgoebel** [comment]
Looks like you need to create a hook which runs `copy_metadata()`, see e.g. PyInstaller/hooks/hook-distutils.py. I'd appreciate a pull-request for the hook :-)

**Comment by nouxu1** [comment]
The same issue. any help please

**Comment by Lokhozt** [comment]
Pull request created #4276

**Comment by bwoodsend** [comment]
Fixed in #4490.
